.github/workflows/scorecards.yml

```diff
@@ -58,14 +59,16 @@ jobs:
       # Upload the results as artifacts (optional). Commenting out will disable uploads of run results in SARIF
       # format to the repository Actions tab.
       - name: "Upload artifact"
-        uses: actions/upload-artifact@65462800fd760344b1a7b4382951275a0abb4808 # v4.3.3
+        uses: actions/upload-artifact@65462800fd760344b1a7b4382951275a0abb4808 # v4.3.1
```

Comment:
Is there a reason you change the version number?

Reply:
I think this was a mistake during rebasing, my appologies!